aem/build.gradle.kts

```diff
@@ -1,10 +1,24 @@
 import com.cognifide.gradle.aem.common.ProgressLogger
 
 plugins {
-    id("com.cognifide.aem.instance")
+    id("com.cognifide.aem.environment")
 }
 
 aem {
+    docker {
```

Comment:
move it to

`aem { environment { docker { `

or 

`aem { config { docker `

as you want

Reply:
done